src/lib/helpers.ts

```diff
@@ -0,0 +1,116 @@
+import { Task } from "gantt-task-react";
```

Comment:
I guess this file is not required as well as the package

Reply:
@joywin2003 Clean this